core/scripts/go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/prometheus/client_golang v1.17.0
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/chainlink-automation v1.0.4
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240618210005-a88f179ffc16
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240619173845-5276747ae1ed
```

Comment:
This is pointing here https://github.com/smartcontractkit/chainlink-common/pull/602 for now

Will merge it in once this PR gets approved